apps/emqx_bridge/src/emqx_action_info.erl

```diff
@@ -130,23 +130,46 @@ action_to_bridge_v1_fixup(ActionOrBridgeType, Config) ->
             Config
     end.
 
-bridge_v1_to_action_fixup(ActionOrBridgeType, Config) ->
+bridge_v1_to_action_fixup(ActionOrBridgeType, Config0) ->
     Module = get_action_info_module(ActionOrBridgeType),
     case erlang:function_exported(Module, bridge_v1_to_action_fixup, 1) of
         true ->
-            Module:bridge_v1_to_action_fixup(Config);
+            Config1 = Module:bridge_v1_to_action_fixup(Config0),
+            common_bridge_v1_to_action_adapter(Config1);
```

Comment:
I think the `common_bridge_v1_to_action_adapter` call should be done before calling the `bridge_v1_to_action_fixup` callback. That way one can always fix things that the "automatic" translation does wrong. What is the reason for doing this after?

Reply:
The reason was that, IMO, it's easier to reason about the transformation for each bridge if one considers only the old and new schemas, rather than something intermediate done by this "automatic" translation, that could change at some point.